patches/net/minecraft/world/entity/EntityType.java.patch

```diff
@@ -33,18 +56,20 @@
 +        FeatureFlagSet p_273518_,
 +        final java.util.function.Predicate<EntityType<?>> trackDeltasSupplier,
 +        final java.util.function.ToIntFunction<EntityType<?>> trackingRangeSupplier,
-+        final java.util.function.ToIntFunction<EntityType<?>> updateIntervalSupplier
++        final java.util.function.ToIntFunction<EntityType<?>> updateIntervalSupplier,
++        final net.neoforged.neoforge.flag.Flag[] requiredFlags
```

Comment:
I'm not a fan of the hidden array->set conversion here. It should accept a Set<T> parameter.